packages/bitcore-lib/lib/transaction/sighash.js

```diff
@@ -97,26 +99,12 @@ var sighash = function sighash(transaction, sighashType, inputNumber, subscript)
  * @param {number} sighash
  * @param {number} inputIndex
  * @param {Script} subscript
- * @param {String} signingMethod - method used to sign - 'ecdsa' or 'schnorr' (future signing method)
  * @return {Signature}
  */
-function sign(transaction, privateKey, sighashType, inputIndex, subscript, signingMethod) {
-  signingMethod = signingMethod || 'ecdsa';
-
+function sign(transaction, privateKey, sighashType, inputIndex, subscript) {
```

Comment:
Consider updating method params to reflect new argument list:

- MultiSigInput.prototype.getSignatures
- MultiSigScriptHashInput.prototype.getSignatures
- PublicKeyInput.prototype.getSignatures
- PublicKeyHashInput.prototype.getSignatures